packages/coinstac-ui/app/render/components/consortium/consortia-list-item.jsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
+import { graphql } from 'react-apollo';
 import PropTypes from 'prop-types';
+import { ConsortiaListWithData } from './consortia-list';
+import { deleteConsortiumByIdFunc, fetchAllConsortiaFunc } from '../../state/graphql-queries';
 import { Button, Panel } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
 
-const ConsortiaListItem = ({ owner, user, consortium }) => (
+const ConsortiaListItem = ({ owner, user, consortium, submit}) => (
   <Panel header={<h3>{consortium.name}</h3>}>
     <p>{consortium.description}</p>
     <LinkContainer to={`/consortia/${consortium.id}`}>
       <Button bsStyle="info">View Details</Button>
     </LinkContainer>
-    {owner && <Button bsStyle="danger" className="pull-right">Delete Consortium</Button>}
+    {owner && <Button bsStyle="danger" onClick={() => submit(`${consortium.id}`)} className="pull-right">Delete Consortium</Button>}
```

Comment:
Sorry to get nitpicky, but would you consider renaming this function from `submit` to `deleteConsortium` or something more relevant to its purpose?